deploy/olm-catalog/ibm-block-csi-operator/1.10.0/manifests/ibm-block-csi-operator.clusterserviceversion.yaml

```diff
@@ -221,6 +221,34 @@ spec:
           - description: The current version of the driver.
             displayName: Version
             path: version
+      - name: hostdefinitions.csi.ibm.com
+        version: v1
+        group: csi.ibm.com
+        kind: HostDefinition
+        displayName: "IBM block storage Host Definition CSI driver"
+        description: "Represents Host Definition for block storage CSI driver"
+        specDescriptors:
+          - description: HostDefinition phase.
+            displayName: HostDefinition phase
+            path: hostdefinition.phase
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:text'
+          - description: HostDefinition HostName In Storage.
+            displayName: HostDefinition Host In Storage
```

Comment:
1. why with and without `Name`?
2. in general, I don't think we should capitalize words that are not names of objects/fields (e.g. `In`/`Storage`), and specifically for this description, since the storage doesn't necessarily have a field called "HostName", I would write something like "The host definition name on storage"

Reply:
done

Reply:
`IBM block storage Host Definition CSI driver` sounds like a CSI driver, not a host definition

Reply:
removed the `CSI driver`, better now?